universal-application-tool-0.0.1/package-lock.json

```diff
@@ -0,0 +1,7845 @@
+{
```

Comment:
[blocking] Delete this file

Reply:
Yes! I will gladly do so. I've always hated the rebase pain this file causes